package/src/types/v1-profile.ts

```diff
@@ -0,0 +1,19 @@
+import { Region } from "./general.js"
```

Comment:
Why are you importing javascript files here?

Also this files doesn't even exist.